hyperion-geiger-counter/src/main/java/com/willowtreeapps/hyperion/geigercounter/DroppedFrameObserver.java

```diff
@@ -0,0 +1,98 @@
+package com.willowtreeapps.hyperion.geigercounter;
+
+import android.annotation.SuppressLint;
+import android.annotation.TargetApi;
+import android.app.Activity;
+import android.content.res.AssetFileDescriptor;
+import android.media.SoundPool;
+import android.util.Log;
+import android.view.Choreographer;
+
+import static android.media.AudioManager.STREAM_SYSTEM;
+import static android.os.Build.VERSION_CODES.JELLY_BEAN;
+import static com.willowtreeapps.hyperion.geigercounter.GeigerCounterPlugin.LOG_TAG;
+
+@TargetApi(DroppedFrameObserver.TARGET_API)
+public class DroppedFrameObserver implements Choreographer.FrameCallback {
+
+    public static final int TARGET_API = JELLY_BEAN;
```

Comment:
I'm not sure that remapping the constant here adds any contextual value. Why not just use `JELLY_BEAN`?

Additionally, as this class actually requires jelly bean and higher `@RequiresAPI` would be more correct and would more naturally push the API level checking on to the calling code.

Reply:
It's used outside this class, and this was a way to have a single source of truth.

Reply:
The class implements `FrameCallback` which was added in Jelly Bean so the entirety of the class requires Jelly Bean and any calls to the class will push the requirement check on the calling code. Because of that, you can actually remove the `TARGET_API` constant as it becomes redundant to the annotation which already specifies it's target api. `@TargetApi` is intended for times when you need to disable the lint check. Imagine a function that internally handles a lint check but does so in a way that lint is unable to infer. Example of a valid target api annotation usage:

```java
@TargetApi(...)
void doSomething() {
  if (isAtLeastJellyBean()) {
     // run some code requiring jelly bean or higher
  } else {
     // run some other code
  }
}
```